webpack.config.js

```diff
@@ -84,9 +84,14 @@ module.exports = {
       { test: /bootstrap-sass/, loader: "imports?jQuery=jquery" },
       { test: /modernizr-latest\.js/,
         loader: "imports?this=>window,html5=>window.html5!exports?window.Modernizr" },
-      { test: /\.json$/,
+      {
+        test: /\.json$/,
         loader: 'json-loader',
-        include: path.join(__dirname, 'node_modules/moment-timezone') },
+        include: [
+            path.join(__dirname, 'node_modules/moment-timezone'),
+            path.join(__dirname, 'troposphere/static/js/mockdata')
```

Comment:
@cdosborn it looks like this is outdated now that we are using `.js` in `mock` dir now?